seeq/addons/mps/_mps_sdl_ui.py

```diff
@@ -672,8 +670,7 @@ def review_ref(self, b):
                                 )
 
         workbook = spy.workbooks.pull(spy.workbooks.search({'ID': self.workbook_id},
-                                                           include_referenced_workbooks=False, include_inventory=False,
-                                                           quiet=True, errors='catalog'
```

Comment:
It could still be useful if you want to have the `include_inventory=False` and `errors='catalog'` when doing either of these workbooks.pull calls